common/calculation/probability/probability.go

```diff
@@ -49,12 +49,15 @@ func probability(m, n, b int64) *big.Rat {
 func calculateBpy(blockTimeRat *big.Rat, pool worker.UtilityVars) *big.Rat {
```

Comment:
"This normalises the BPY by token decimals, this returns the BPY in USD, where previously it was USD scaled to token decimals" (needs @cibo6 )